chsdi/tests/integration/__init__.py

```diff
@@ -32,6 +35,19 @@ def tearDown(self):
         del self.testapp
         del self.grids
 
+    @contextmanager
+    def getSession(self):
+        session = scoped_session(sessionmaker())
+        yield session
+        session.close()
+
+    def getRandomFeatureId(self, bodId):
+        models = models_from_bodid(bodId)
+        with self.getSession() as session:
+            query = session.query(models[0]).limit(1)
+            reslt = query.one()
```

Comment:
```sql
SELECT sub.id FROM (SELECT id FROM table LIMIT 1000) AS sub ORDER BY RANDOM() LIMIT 1
```

The first 1000 will always be the same, and we choose randomly one out of a 1000